pkg/agent/billing.go

```diff
@@ -196,9 +198,11 @@ func (h *vmMetricsHistory) finalizeCurrentTimeSlice() {
 	// TODO: This approach is imperfect. Floating-point math is probably *fine*, but really not
 	// something we want to rely on. A "proper" solution is a lot of work, but long-term valuable.
 	metricsSeconds := vmMetricsSeconds{
-		cpu: uint32(math.Round(float64(h.lastSlice.metrics.cpu) * seconds)),
+		cpu:        uint32(math.Round(float64(h.lastSlice.metrics.cpu) * seconds)),
```

Comment:
`math.Round` goes away from zero, we might want to to make this `math.Floor` eventually to round in users' favor.

Reply:
Good point. I don't know — see also: https://neondb.slack.com/archives/C044R76J3T5/p1677771940715709

Leaving this comment up for now, until we make a decision + changes

Reply:
Made a change from floor -> round for activeTime as well. Otherwise we send 29 or 30 seconds, basically missing 1/60 of active time due to clock stuff